README.md

```diff
@@ -32,3 +32,7 @@ The is the file format produced by the GUID Tool: one line per subject in the fo
 
 ## Example outputs
 See [/examples](/examples)
+
+## Notes:
+Column experiment_id must be manually filled in for now.
+This is based on experiment ID's received from NDA after setting the study up through the NDA website [here](https://ndar.nih.gov/user/dashboard/collections.html).
```

Comment:
am I reading it right @mtnhuck that it is pretty much yet another "mapping" but now for the task/runs?  unfortunately that url requires a login so can't assess.  Could you please give an example?

Reply:
This is a number that comes from the NDA archive, ie they provide you with a number for your experiment. I'm still waiting to hear back from them if it is at the session level or the scan level (ie would resting-state need a different experiment number than a face-perception task?). An example number which I generated on their website is 867.

Reply:
Ok, let's wait for them to reply and then make it an option or mapping

Reply:
According to the .gov ppl, `experiment_id` is per "experiment within the study" so resting state would get one and face-perception would get another.

Reply:
As agreed later on, this is the id for user to provide. We have a preliminary patch ready to submit (will be done after this PR is merged) to provide the mapping from BIDS task id to experiment_id via a simple .tsv file.